docs/cn_omf.md

```diff
@@ -201,18 +214,150 @@ Bloom Header
 
 This is followed by the bloom filter.
 
-### HyperLogLog
+#### HyperLogLog
+
+This region does not have a separate header, just the hyperloglog (hlog) bytes.
+The offset and length in the kblock header points to this region. Each kblock
+stores its hlog bytes and the kvset's hlog can be composed using the individual
+kblock hlogs.
+
+### Vblock (Value block)
+
+VBlocks consist of values followed by a trailer. The offset of a specific value
+and its length are obtained from the kmd entry for this value which is linked to
+the corresponding key.
+
+#### Vblock Trailer
+
+Placed at the end of vblock - 4K bytes.
+
+    +---------------------+
+    | u32 vbh_magic       |
+    | u32 vbh_version     |
+    | u32 vbh_min_key_off | Offset of the min key
+    | u32 vbh_min_key_len | Length of the min key
+    | u32 vbh_max_key_off | Offset of the max key
+    | u32 vbh_max_key_len | Length of the max key
+    |---------------------|
+    | min_key             | 1344 bytes is reserved at all times
+    |---------------------|
+    | max_key             | 1344 bytes is reserved at all times
+    +---------------------+
+
+- `min_key` is the smallest key that references data within this vblock
+- `max_key` is the largest key that references data within this vblock
+
+These are used to determine which vblocks will be affected by a kvset split.
+
+### Hblock (Kvset header)
+
+Mblock consisting of kvset metadata.
 
-This region does not have a separate header, just the hyperloglog (hlog) bytes.  The offset and length in the KBlock header points to this region.  Each kblock stores its hlog bytes and the kvset's hlog can be composed using the individual kblock hlogs.
+    +----------+--------+-------+----------+
+    |          |        |       |          |
+    | Metadata | Vblock | PTree | Hyperlog |
+    |  Header  | Index  |       |          |
+    |          | Adjust |       |          |
+    +----------+--------+-------+----------+
 
-## VBlocks (Value blocks)
+#### Metadata Header
```

Comment:
How about storing the min/max key of a kvset in the kvset metadata header? I don't know if there are plans to track this somewhere else. I think storing the min/max will be useful to rehydrate the edge map at cN open.

Reply:
@alexttx I can't remember if we discussed this or not. I feel like I remember this coming, but I can't remember the solution we went with.

cc @gsramdasi

Reply:
currently each kblock stores its min/max keys.  so we can easily get the min/max key of the kvset by looking at the first/last kblocks.  i might not bother tracking in the kvset header initially.